prysmaticlabs/prysm
# Moving slow tasks from Travis to Buildkite

This moves more of the work to Buildkite.

At this point, both platforms should be required for merge.

Travis does:
- Code coverage reporting
- Solidity linting 
- Gometalinter

Buildkite does:
- Build everything 
- Test everything
- Check BUILD visibility 
- Check TODO format
- Check gazelle

No one does:
- Build for Mac OS X

We should probably consider adding a Mac OS X runner to the pipeline in the future, but for now we won't be changing any of the underlying C++ dependencies so this is unlikely to be an issue.
